regolith/database.xsh

```diff
@@ -28,12 +30,15 @@ def load_git_database(db, client, rc):
     dbdir = dbdirname(db, rc)
     # get or update the database
     if os.path.isdir(dbdir):
-        cmd = ['git', 'pull']
-        cwd = dbdir
+        with indir(dbdir):
+            git pull upstream master or git pull origin master or git pull @(rc.remote) master or git pull @(rc.remote) @(rc.branch) or git pull
```

Comment:
@scopatz what is the best way to get this over multiple lines?

Reply:
backslash at the end of the line (line continuation)

Reply:
Ah ok I was trying to do 
```
(git pull upstream master
 or git pull origin master ...)
```

Reply:
Or you could use `()` and explicit subprocesses, eg. `(![git pull upstream master] or\n ![git pull origin master])`

Reply:
Yeah, you can, you just need to be explicit about it, I think

Reply:
Xonsh seems not happy about
```
git pull upstream master \
            or git pull origin master \
            or git pull @(rc.remote) master \
            or git pull @(rc.remote) @(rc.branch) \
            or git pull
```

Reply:
Is the backslash the last char  on the line (no whitespace)?

Reply:
Yes

Reply:
hmmm please report on the xonsh issue tracker and use explicit subprocs instead. Thanks!